src/victory-primitives/clip-path.js

```diff
@@ -28,28 +29,25 @@ export default class ClipPath extends React.Component {
     translateY: PropTypes.number
   };
 
+  static defaultProps = {
+    translateX: 0,
```

Comment:
I'm not quite sure what you mean. Parent components are taking default props of their children?